src/python/dxpy/scripts/dx.py

```diff
@@ -3823,17 +3829,26 @@ def get_archival_paths(args):
         
         paths = [split_unescaped(':', path, include_empty_strings=True) for path in args.path]
         possible_projects = set()
-        possible_objects = set()
+        possible_folder = set()
+        possible_files = set()
+
+        # Step 0: parse input paths into projects and objects
         for p in paths:
             if len(p)>2: 
-                err_exit("Path '{}' is invalid. Please check the inputs or check --help for example inputs.".format(p), code=3)
+                err_exit("Path '{}' is invalid. Please check the inputs or check --help for example inputs.".format(":".join(p)), code=3)
             elif len(p) == 2:
                 possible_projects.add(p[0])
             elif len(p) == 1:
                 possible_projects.add('')
             
-            possible_objects.add(p[-1])
+            obj = p[-1]
+            if obj[-1] == '/':
```

Comment:
Can you make resolution of path work in way that `dx ls` resolves it's paths? 
```
$ dx ls -la x\\/
closed  2021-08-12 15:23:39 0 bytes   x/ (file-G4BYkyQ0Fgk8672vB7fKjqPJ)
$ dx archive x\\/
path ['x\\/']
Failed request: {'folder': '/', 'recurse': True, 'allCopies': False}
```

Reply:
```
➜  dx archive x\\/       
Will tag 1 file(s) for archival in project-G4B26p005pGF60Jj4K5vf8Q0
Confirm all paths? [Y/n]: n
➜  dx archive x/  
Will tag file(s) for archival in folder project-G4B26p005pGF60Jj4K5vf8Q0:/x recursively
Confirm all paths? [Y/n]: n
```